yaml/ver10/cloudintegration/yaml/cloudintegration.yaml

```diff
@@ -0,0 +1,149 @@
+openapi: 3.1.0
+info:
+  title: Cloud Integration Specification
+  contact:
+    name: ONVIF Technical Specification
+    url: https://github.com/onvif/specs
+    email: help@onvif.org
+  license:
+    name: ONVIF Contributor License Agreement
+    url: https://github.com/onvif/specs/blob/development/LICENSE.md
+  version: "25.06"
+
+components:
+  schemas:
+    Base64DERencodedASN1Value:
+      description: This schema is the equivalent of tas:Base64DERencodedASN1Value in the Security Service specification.
+      type: string
+      format: byte
+      examples: [ Base64DERencodedASN1Value ]
+
+    EndpointReferenceType:
+      description: This schema is the equivalent of wsa:EndpointReferenceType in the WA-Addressing 1.0 specification.
+      type: string
+      format: uri
+
+    AuthorizationServerConfiguration:
+      type: object
+      description: This schema is the equivalent of tas:AuthorizationServerConfigurationData
+      properties:
+        ServerURI:
+          description: Authorization server address
+          type: string
+        ClientID:
+          description: Client identifier issued by the authorization server
+          type: string
+        Scope:
+          description: The requested access scope
+          type: string
+      required:
+        - ServerURI
+        - ClientID
+
+    UplinkConfiguration:
+      type: object
+      properties:
+        RemoteAddress:
+          description: Uniform resource locator by which the remote client can be reached.
+          item:
+            $ref: '#/components/schemas/EndpointReferenceType'
+        UserLevel:
+          description: Authorization level that will be assigned to the uplink connection.
+          type: string
+          enum: [Administrator, Operator, User]
+        UplinkTrustAnchor:
+          items:
+            $ref: '#/components/schemas/Base64DERencodedASN1Value'
+          description: The certificate with the public key used to validate the TLS certificate of the uplink server.
+      required:
+        - RemoteAddress
+        - UserLevel
+
+    DeviceConfiguration:
+      type: object
+      properties:
+        ShareToken:
+          description: The share token, to prove permission to share the device.
+          item:
+            type: string
+            format: byte
+        EndpointReferences:
```

Comment:
Here endpoint reference is passed for each device separately. But in Devicesharing completed notification Device information is passed as Array. In my opinion either endpoint reference should be single or callback data devicesinformation should be single to make it consistent.

Reply:
Ok. I'd make one call per device. Let's discuss in the WG

Reply:
I made it a single object and not an array in deviceSharingCompleted